data-processed/Karlen-pypm/metadata-Karlen-pypm.txt

```diff
@@ -5,7 +5,7 @@ model_contributors: Dean Karlen (University of Victoria and TRIUMF) <karlen@uvic
 website_url: https://pypm.github.io/home/
 license: gpl-3.0
 team_model_designation: primary
-methods: Discrete-time difference equations with long periods of constant transmission rate
+methods: Discrete-time difference equations with long periods of constant transmission rate. Jan 31 2021: include vaccination. Feb 28 2021: added a variant strain
```

Comment:
I think there is an issue having a full colon in YAML files.

Reply:
Thanks for the hint! The validation code that detected the error, has a bug, so it crashed on its own.

Reply:
Yes, apologies for that. It is fixed now.